src/footer/template.njk

```diff
@@ -0,0 +1,79 @@
+<footer class="govuk-c-footer {{ params.classes if params.classes }}"
+        {% for attribute, value in params.attributes %} {{attribute}}="{{value}}"{% endfor %}>
+  <div class="govuk-o-width-container {{ params.containerClasses if params.containerClasses }}">
+    {% if params.navigation %}
+      <div class="govuk-c-footer__navigation">
+        {% for item in params.navigation %}
+          <div class="govuk-c-footer__section">
+            <h2 class="govuk-c-footer__heading govuk-heading-m">{{ item.title }}</h2>
+            {% if item.items %}
+              {% set listClasses %}
+                {% if item.columns %}
+                  govuk-c-footer__list--columns-{{ item.columns }}
+                {% endif %}
+              {% endset %}
+              <ul class="govuk-c-footer__list {{ listClasses | trim }}">
+                {% for item in item.items %}
+                  {% if item.href and item.text %}
+                    <li class="govuk-c-footer__list-item">
+                      <a class="govuk-c-footer__link" href="{{ item.href }}">
+                        {{ item.text }}
+                      </a>
+                    </li>
+                  {% endif %}
+                {% endfor %}
+              </ul>
+            {% endif %}
+          </div>
+        {% endfor %}
+      </div>
+    {% endif %}
+    <div class="govuk-c-footer__meta">
+      <div class="govuk-c-footer__meta-item govuk-c-footer__meta-item--grow">
+        {% if params.meta and params.meta.items %}
+          <h2 class="govuk-h-visually-hidden">Support links</h2>
+          <ul class="govuk-c-footer__inline-list">
+            {% for item in params.meta.items %}
+              <li class="govuk-c-footer__inline-list-item">
+                {% if item.href %}
+                  <a class="govuk-c-footer__link" href="{{ item.href }}">
+                    {{ item.text }}
+                  </a>
+                {% elseif item.html %}
+                  {{ item.html | safe }}
+                {% endif %}
+              </li>
+            {% endfor %}
+          </ul>
+        {% endif %}
+        <svg
```

Comment:
Worth trying to see if this can be optimised – a quick play with http://petercollingridge.appspot.com/svg-optimiser suggests we can shave a few hundred bytes off this.

Reply:
Yeah I've already optimised it, those bytes be from putting it on a single line which I figured is whitespace that'll get GZIPed but I'll double check.

Reply:
👍

Reply:
Confirmed the extra savings come from the whitespace